x-pack/plugins/enterprise_search/public/applications/shared/layout/nav.tsx

```diff
@@ -154,25 +160,29 @@ export const useEnterpriseSearchNav = (alwaysReturn = false) => {
         defaultMessage: 'Build',
       }),
     },
-    {
-      id: 'relevance',
-      items: [
-        {
-          id: 'inference_endpoints',
-          name: i18n.translate('xpack.enterpriseSearch.nav.inferenceEndpointsTitle', {
-            defaultMessage: 'Inference Endpoints',
-          }),
-          ...generateNavLink({
-            shouldNotCreateHref: true,
-            shouldShowActiveForSubroutes: true,
-            to: INFERENCE_ENDPOINTS_PLUGIN.URL + INFERENCE_ENDPOINTS_PATH,
-          }),
-        },
-      ],
-      name: i18n.translate('xpack.enterpriseSearch.nav.relevanceTitle', {
-        defaultMessage: 'Relevance',
-      }),
-    },
+    ...(hasEnterpriseLicense
+      ? [
+          {
+            id: 'relevance',
```

Comment:
can you add this item to the new project navigation, it will need to be added to [getNavigationTreeDefinition](https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/navigation_tree.ts#L68). You can look at the testing instructions in #186429 or #186178 to test with the new Solution Nav.

Reply:
Added to the new solution nav